src/ServicePulse.Host/app/js/app.constants.js

```diff
@@ -6,7 +6,7 @@
         .constant('scConfig', {
             default_route: '/dashboard',
             service_control_url: 'http://localhost:33333/api/',
-            monitoring_urls: ['http://localhost:33633/']
+            monitoring_urls: ['http://localhost:33533/']
```

Comment:
What is the reason to change this URL? is it only local change?

Reply:
Not meant to be changed, I probably forgot to revert the change before commiting.